docs/release-notes.txt

```diff
@@ -6,6 +6,35 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.4
+^^^^^^^^^^^^^^
+
+**Release Date:** May 14, 2020
+
+- **Breaking Change:** Users are no longer automatically logged in as the "determined" user. Refer to the :ref:`users` documentation for more details.
+
+- Support multi-slot notebooks. The number of slots per notebook cannot exceed the size of largest available agent. 
+
+- Authentication sessions now expire every 7 days.
```

Comment:
Perhaps "after"? It's not exactly a periodic event.

Reply:
lol true